initrd/init

```diff
@@ -5,6 +5,9 @@ echo "hello world" > /dev/ttyprintk
 # Setup our path
 export PATH=/sbin:/bin:/usr/sbin:/usr/bin:/usr/local/bin
 
+#export GPG_TTY STATICALLY (NEED TO FIX)
+export GPG_TTY=/dev/console
```

Comment:
```suggestion
export GPG_TTY=$(tty)
```
tty is installed, so it could be used here.  tty returns /dev/console, so that doesn't change.

Reply:
@jandryuk : I've took that approach initially. Exiting to recovery shell consequently gives:
```
echo $GPG_TTY
not a tty

tty
/dev/console
```

init script doesn't have an access to a TTY.
This is why gpg command verifying signatures  is `gpg --no-tty`, because from a whiptail perspective, it doesn't have access to a shell until ash is spawned. The reason why gpg signing works is because pinentry-tty is called for which requires the tty to be set, else it also fails.

I haven't found a more elegant solution then to hardcode the tty, force the paths to binaries in config and force gpg-agent usage. If you have a better solution, please advise. :)

Reply:
I'm pretty sure I used `$(tty)` for qemu's recovery shell and it worked. But that wasn't the whiptail stuff.  Can you run the whiptail stuff under qemu?

Anyway, I don't have a better suggestion.

Reply:
Oh, your call to `tty` was before /dev/ was mounted.  Setting GPG_TTY later works for me in QEMU.  Check out this commit: https://github.com/jandryuk/heads/commit/ab8046b5cc51b8c9863c639ee3d614c144723f3d

Reply:
@jandryuk if you want to test fbwhiptail/gui-init, see this branch: https://github.com/flammit/heads/tree/qemu-gui-init.  Feel free to drop by u-root.slack.com / #heads channel if you wanna chat live (invite available from slack.u-root.com)

Reply:
@flammit Oh, I forgot to thank you for the gui branch.  Thanks!